manual.asc

```diff
@@ -80,17 +80,27 @@ Note that the currently selected deck is highlighted with a grey background, and
 === Other Deck Actions
 Long tapping on a deck will show a list of other actions available to perform on that deck:
 
-Rename deck :: Use this option to rename a deck
+Add :: Use this option to create a new empty note to the seleted deck
+
+Browse cards :: Use this option to see the cards of the selected deck with the <<browser, card browser>>
+
+Rename deck :: Use this option to rename the selected deck
+
+Create subdeck :: Use this option to make a https://docs.ankiweb.net/getting-started.html#decks[subdeck] of the selected deck
 
 Deck options :: Tapping on deck options allows you to configure various deck specific study options.
 Please see the https://docs.ankiweb.net/deck-options.html#deck-options[desktop documentation] for more information about these study options.
 
 Custom study :: Allows you to choose from some convenient presets for studying outside of your normal schedule, for example increasing the study limit for the day. See the section on <<filtered,"filtered decks">> for more detailed information.
 
-Delete deck :: Use this option to delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
-
 Export deck :: This option can be used to share a deck with other users. See the <<exporting, exporting decks>> section for more information.
 
+Create shortcut :: Allows you to add a shortcut icon of the selected deck to the home screen of the system (Android)
+
+Edit description :: Use this option to create or edit additional text that is shown on the <<deckOverview, deck overview>> screen of the selected deck
+
+Delete deck :: Use this option to delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
```

Comment:
```suggestion
Delete deck :: Delete a deck (note: this action is not reversible, although you can <<backups,restore from a backup>>)
```

Reply:
Is this true? Anki's undo should work.

Reply:
This isn't true now. It is a part of text in the current manual, and it should be fixed. 

Though I didn't write this part (I just changed the order of the item), I overlooked it. Thank you for pointing it out.

Reply:
How about this change:
`Delete deck :: Deletes the selected deck. (You can undo this action by 'Undo' button in the popup message shown after the action, or 'Undo Delete Deck' in the screen menu.)`

Reply:
That's great!